### ISSUE TITLE
Add WAITING state

### ISSUE THREAD

**Issue by alensiljak** [orgzly-revived/orgzly-android-revived#89]
As per GTD, there is also the Waiting state for when the task depends on an external party.
I'd suggest that we add it to the default list (todo, next, done).
Not sure what it is like in OrgMode in Emacs but I find that one really helpful.

**Comment by amberin** [comment]
The idea is that you customize your TODO keywords in the app settings, according to your needs.

**Comment by alensiljak** [comment]
> The idea is that you customize your TODO keywords in the app settings, according to your needs.

Right. I get that. However, after reinstalling the app, these need to be set up manually again and the app warns that "the files will be modified" after applying so I have some concerns. Will need to check which exact changes that does to the files, though.

However, the question is a bit different. The two basic states for a TODO app are TODO and DONE. In Orgzly, on the other hand, there is also the NEXT state, hinting that the principles are borrowed from GTD (it's no secret that this is a great tool to help using that methodology).

There, the terms NEXT and WAITING are coming from:

![image](https://github.com/orgzly-revived/orgzly-android-revived/assets/462445/a2939d21-3d5d-433e-a04c-9ce5ebb19e7d)

So, we have NEXT but no WAITING, which is really helpful for delegated items and causes no additional issues.
Or, rather, whichever arguments apply for NEXT might also apply for WAITING. Unless I'm missing something.

**Comment by stefan2904** [comment]
I also use the WAITING state and agree that it is simple to configure it via the custom states, as it has no special meaning to Orgzly (its simply another "open" state which can be filtered in agendas etc).

> Right. I get that. However, after reinstalling the app, these need to be set up manually again

This is true for all configuration settings, right? I agree that this is annoying when you re-install Orgzly (or switch from the original to revived), so a more sustainable feature could be to export/import the settings (or even allow to sync them alongside the org files). 

> the app warns that "the files will be modified" after applying so I have some concerns. 

To my knowledge, this only re-initializes the orgzly database with the new todo states, and not changing any content. 

> So, we have NEXT but no WAITING, which is really helpful for delegated items and causes no additional issues.

I think it is subjective what states are there per default. Personally, I don't have any opinion whether WAITING should be there per default, since I configure to my needs anyway. But it is probably a good idea to not have too many states per default.

**Comment by amberin** [comment]
I don't have strong opinions either way, but fwiw I use a completely different set of states even though I consider myself to be following GTD pretty closely. I've always thought that which states make sense depend very much on what type of tasks you are dealing with. In the case of "next", I prefer using a tag for that.

**Comment by chaoflow** [comment]
Some others and I prefer WAIT for the waiting state, as it has 4 letters as well and therefore aligns nicely with "TODO" and "DONE", though personally I don't use "TODO" but configure differently.

For the sake of this issue I suggest to add WAIT to the default or close as won't fix.

Once we have #128 we could define pre-sets that can be loaded.
